hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/ozoneimpl/ContainerScanHelper.java

```diff
@@ -131,6 +132,23 @@ public void handleUnhealthyScanResult(long containerID, ScanResult result) throw
     if (containerMarkedUnhealthy) {
       metrics.incNumUnHealthyContainers();
     }
+    // triggering a volume scan for the unhealthy container
+    triggerVolumeScan(containerData);
```

Comment:
I'm thinking we should only do this if `containerMarkedUnhealthy == true`. With reconciliation we will now scan containers that were already unhealthy. I don't think we need new volume scans each time the same unhealthy container shows up.